### DISCUSSION TITLE
Will it have a user ranking system?

### DISCUSSION THREAD

**Discussion by edsonweslley** [monuelo/i-just-want-to-play-tetris#25] [Q&A] [answered]
I was curious about adding a ranking system.

**Comment by 0xHericles** [comment, marked as answer]
Hello, @edsonweslley! Yes, it's on our plans. There's already an open issue for that, to allow players to save their scores in the leaderboard with theirs names or initials, maybe. 🏆🥇 

Here's the open issue: 
- [Create leaderboard #6](https://github.com/monuelo/i-just-want-to-play-tetris/issues/6)

And you also might be interested in this one: https://github.com/monuelo/i-just-want-to-play-tetris/issues/7. 🧑🏽‍🤝‍🧑🏽
